### ISSUE TITLE
Big Data: Paired-end analysis

### ISSUE THREAD

**Issue by rahulnccs** [benjjneb/dada2#405]
i'm running DADA2 for the first time and I'm getting "Forward and reverse files do not match" as a output 
for:
if(!identical(sample.names, sample.namesR)) stop("Forward and reverse files do not match.")

am I going right?

Thanks in advance

**Comment by benjjneb** [comment]
Looks like not. The script does not think your forward and reverse files have matched names.

What is the file format of your sequence files? Does it match what's expected in the big data workflow?

**Comment by rahulnccs** [comment]
file were fastq.gz and has the same prefix for eg.
samplename_F_filt.fastq.gz for Forward read and 
samplename_R_filt.fastq.gz for reverse one.

As mentioned previously I got an error as "Forward and reverse files do not match" but ignoring that, when I executed subsequent script for dereplication, got output like this:
Processing: CS211F 
Sample 1 - 680609 reads in 208861 unique sequences.

Sample 1 - 680609 reads in 185732 unique sequences.

Duplicate sequences in merged output.
Processing: CS213F 
Sample 1 - 674496 reads in 188719 unique sequences.

Sample 1 - 674496 reads in 176819 unique sequences.
